substrait_consumer/tests/functional/relations/join_relation_snapshots/DataFusionProducer/right_anti_join_plan.json

```diff
@@ -0,0 +1,123 @@
+        "name": "equal"
+      }
+    }
+  ],
+  "relations": [
+    {
+      "root": {
+        "input": {
+          "join": {
+            "left": {
+              "read": {
+                "baseSchema": {
+                  "names": [
+                    "o_orderkey",
+                    "o_custkey",
+                    "o_orderstatus",
+                    "o_totalprice",
+                    "o_orderdate",
+                    "o_orderpriority",
+                    "o_clerk",
+                    "o_shippriority",
+                    "o_comment"
+                  ]
+                },
+                "projection": {
+                  "select": {
+                    "structItems": [
+                      {},
+                      {
+                        "field": 1
+                      }
+                    ]
+                  }
+                },
+                "namedTable": {
+                  "names": [
+                    "orders_small"
+                  ]
+                }
+              }
+            },
+            "right": {
+              "read": {
+                "baseSchema": {
+                  "names": [
+                    "l_orderkey",
+                    "l_partkey",
+                    "l_suppkey",
+                    "l_linenumber",
+                    "l_quantity",
+                    "l_extendedprice",
+                    "l_discount",
+                    "l_tax",
+                    "l_returnflag",
+                    "l_linestatus",
+                    "l_shipdate",
+                    "l_commitdate",
+                    "l_receiptdate",
+                    "l_shipinstruct",
+                    "l_shipmode",
+                    "l_comment"
+                  ]
+                },
+                "projection": {
+                  "select": {
+                    "structItems": [
+                      {}
+                    ]
+                  }
+                },
+                "namedTable": {
+                  "names": [
+                    "lineitem_small"
+                  ]
+                }
+              }
+            },
+            "expression": {
+              "scalarFunction": {
+                "arguments": [
+                  {
+                    "value": {
+                      "selection": {
+                        "directReference": {
+                          "structField": {}
+                        }
+                      }
+                    }
+                  },
+                  {
+                    "value": {
+                      "selection": {
+                        "directReference": {
+                          "structField": {
+                            "field": 2
+                          }
+                        }
+                      }
+                    }
+                  }
+                ]
+              }
+            },
+            "type": "JOIN_TYPE_ANTI"
```

Comment:
We recently added JOIN_TYPE_RIGHT_ANTI.

Reply:
i'll file a ticket with datafusion to get this updated.

Reply:
They will need to use the new name (or choose LEFT_ANTI) once they upgrade their Substrait version.  Same for other join types.